test/crowdin-fetch.spec.js

```diff
@@ -0,0 +1,45 @@
+const fetch = require('jest-fetch-mock');
+jest.setMock('node-fetch', fetch);
+jest.mock('fs');
+
+const crowdinFetch = require('../lib/crowdin-fetch');
+
+describe('crowdin-fetch', () => {
+  const sampleResponse = {
+    "success": {
+        "status": "skipped"
+    }
+  };
+
+  beforeEach(() => {
+    fetch.resetMocks();
+  });
+
+  test('crowdinFetch calls node-fetch, sends FormData and gets sample JSON response', async () => {
+    fetch.mockResponse(JSON.stringify(sampleResponse));
```

Comment:
Should it be `mockResponseOnce`?

Reply:
indeed, also moved it to beforeEach section